src/components/sidebar/components/sidebar-back-to-link/index.tsx

```diff
@@ -21,42 +13,14 @@ const SidebarBackToLink = ({
   text,
   url,
 }: SidebarBackToLinkProps): ReactElement => {
-  if (!text || !url) {
-    return <PlaceholderBackToLink />
-  }
-
-  return (
-    <MaybeInternalLink className={s.backToLink} href={url}>
-      <IconArrowLeft16 className={s.icon} />
-      <BackToLinkText>{text}</BackToLinkText>
-    </MaybeInternalLink>
-  )
-}
-
-const PlaceholderBackToLink = (): ReactElement => {
   return (
-    <DevPopover
-      buttonClassName={s.popoverButton}
-      title="Work in progress"
-      note={
-        <>
-          We are still ironing out the functionality of this component and how
-          it should behave on the various subpages for each product.
-          <br />
-          <br />
-          We&apos;re also working on UI polish. You can{' '}
-          <a href="https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Dev-Portal?node-id=1498%3A43240">
-            view the revised designs in Figma
-          </a>
-          .
-        </>
-      }
-    >
-      <div className={s.backToLink}>
-        <IconArrowLeft16 className={s.icon} />
-        <BackToLinkText>Back to [Page]</BackToLinkText>
-      </div>
-    </DevPopover>
+    <StandaloneLink
+      className={s.root}
```

Comment:
Blocking: I think we might need a wrapper element with `s.root` applied - otherwise, I think this is causing `margin` & `padding` issues on `:focus`:

![CleanShot 2022-04-13 at 09 28 09](https://user-images.githubusercontent.com/4624598/163191191-ec958f3a-e05e-48f7-940b-ee36884df233.gif)

Reply:
I need to make some spacing adjustments after some convos yesterday anyways, I'm going to see if a different approach helps resolve this. 🤔  Great catch!